hnn_core/dipole.py

```diff
@@ -30,6 +30,8 @@ def simulate_dipole(net, n_trials=None, record_vsoma=False):
         net.params['N_trials'] will be used
     record_vsoma : bool
         Option to record somatic voltages from cells
+    postproc : bool
```

Comment:
Here is the other conflicting spot. I think post-processing should be listed first so the docstring would become
```
postproc : bool
     If False, no postprocessing applied to the dipole
record_vsoma : bool
     Option to record somatic voltages from cells
record_isoma : bool
     Option to record somatic currents from cells
```

Reply:
I'm fine with `postproc` being listed last as long as it matches the function signature. You have less chance of breaking existing user code when you add an argument to the end of the function. E.g., if someone did:

```py
record_vsoma = True
simulate_dipole(net, n_trials, record_vsoma, record_isoma)
```

now if you add `postproc` in the middle, it will get passed as `record_vsoma` when the user intended otherwise